blocksuite/blocks/src/_common/adapters/notion-html/notion-html.ts

```diff
@@ -112,11 +112,17 @@ export class NotionHtmlAdapter extends BaseAdapter<NotionHtml> {
 
   deltaConverter: NotionHtmlDeltaConverter;
 
-  constructor(
-    job: Job,
-    readonly blockMatchers: BlockNotionHtmlAdapterMatcher[] = defaultBlockNotionHtmlAdapterMatchers
-  ) {
+  readonly blockMatchers: BlockNotionHtmlAdapterMatcher[];
+
+  constructor(job: Job, provider: ServiceProvider) {
     super(job);
+    const blockMatchers = Array.from(
```

Comment:
Please also move this file to affine-shared

Reply:
next pr